en/components/stepper.md

```diff
@@ -0,0 +1,82 @@
+
+<img class="responsive-img" src="../images/stepper_horizontal.png" srcset="../images/stepper_horizontal@2x.png 2x" />
+<img class="responsive-img" src="../images/stepper_horizontal_content-off.png" srcset="../images/stepper_horizontal_content-off@2x.png 2x" />
+
+## Steps
+
+Each Step consists of Indicator and Information. The Indicator comes with two types - Number and Icon. You can switch between the two using the "Type" property from the properties panel. You can also choose whether you want to display only Indicator or Information. To do so, you can simply hide them from the layers panel. You also have the option to show/hide the Subtitle by swithing on/off the Subtitle boolean property of the nested Step component.
+
+<img class="responsive-img" src="../images/step_number.png" srcset="../images/step_number@2x.png 2x" />
+<img class="responsive-img" src="../images/step_icon.png" srcset="../images/step_icon@2x.png 2x" />
+
+We also provide a number of states for the Step, which should cover all use case scenarios. These include: Complete, Incomplete, Disabled, Optional, Error and Current. These can be switched from the "State" property on the properties panel. We also provide two additional states: Hover and Focused, which can be used by switching on/off the boolean properties "Hover" and "Focused". 
+
+<img class="responsive-img" src="../images/step_state.png" srcset="../images/step_state@2x.png 2x" />
+<img class="responsive-img" src="../images/step_state2.png" srcset="../images/step_state2@2x.png 2x" />
+
+We also provide two types of Progress Lines - Active and Inactive. These can be swapped by using the boolean property "Value" from the properties panel, after selecting the nested Progress Line component.
+
+<img class="responsive-img" src="../images/stepper_progress-line_active.png" srcset="../images/stepper_progress-line_active@2x.png 2x" />
+<img class="responsive-img" src="../images/stepper_progress-line_inactive.png" srcset="../images/stepper_progress-line_inactive@2x.png 2x" />
+
+## Content
+
+As mentioned above, When a Step is set to "Current" state, a frame called "Content" is displayed underneath the current Step on the vertical Stepper and underneath all steps on the horizontal Stepper. By default, the Content comes with a simple text layer, where you can add information about the current step. If you'd like to cusomize it in some way, you'll need to detach the component by right-clicking on it and selecting "Detach instance". You can also fully hide it by using the "Content" property on the horizontal Stepper or by going into the nested Step component and switching off the boolean property "Body" on the vertical Stepper.
```

Comment:
"Current" -> Active 
Underneath the Header within the current Step?

Reply:
Fixed.